src/main/resources/view/PersonListCard.fxml

```diff
@@ -10,12 +10,14 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.scene.shape.Rectangle?>
 
 <HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1">
   <GridPane HBox.hgrow="ALWAYS">
     <columnConstraints>
       <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
     </columnConstraints>
+      <Rectangle fx:id="flag" fill="#fa202000" height="120.0" width="6.0" />
```

Comment:
Have you considered using a `pane` instead? In that way, you don't need to set the height. It would be better if the height is `computed-size` instead of fixing one. This would give a cleaner look, even if the `PersonListCard` itself expands. Then the colour can be modified using `-fx-background-colour`

Reply:
These would be the edit required in you want to change:
In `PersonCard`:
```java 
@FXML
    private Pane flag;

if (person.getFlag().isFlagged) {
    flag.setStyle("-fx-background-color: red");
}
```

In `fxml` file: 
```java 
<Pane fx:id="flag" prefWidth="10.0" />
```

Reply:
Oh nice! I'll make the change, thanks for the recommendation.